common/reducers/config/nodes/staticNodes.ts

```diff
@@ -1,191 +1,36 @@
-    network: 'Kovan',
-    isCustom: false,
-    service: 'AUTO',
-    lib: shepherdProvider,
-    estimateGas: false
-  },
-  kov_ethscan: {
-    network: 'Kovan',
-    isCustom: false,
-    service: 'Etherscan.io',
-    lib: shepherdProvider,
-    estimateGas: false
-  },
-
-  rin_auto: {
-    network: 'Rinkeby',
-    isCustom: false,
-    service: 'AUTO',
-    lib: shepherdProvider,
-    estimateGas: false
-  },
-  rin_ethscan: {
-    network: 'Rinkeby',
-    isCustom: false,
-    service: 'Etherscan.io',
-    lib: shepherdProvider,
-    estimateGas: false
-  },
-  rin_infura: {
-    network: 'Rinkeby',
-    isCustom: false,
-    service: 'infura.io',
-    lib: shepherdProvider,
-    estimateGas: false
-  },
-
-  etc_auto: {
-    network: 'ETC',
-    isCustom: false,
-    service: 'AUTO',
-    lib: shepherdProvider,
-    estimateGas: false
-  },
-  etc_epool: {
-    network: 'ETC',
-    isCustom: false,
-    service: 'Epool.io',
-    lib: shepherdProvider,
-    estimateGas: false
-  },
-  etc_commonwealth: {
-    network: 'ETC',
-    isCustom: false,
-    service: 'Ethereum Commonwealth',
-    lib: shepherdProvider,
-    estimateGas: false
-  },
-
-  ubq_auto: {
-    network: 'UBQ',
-    isCustom: false,
-    service: 'AUTO',
-    lib: shepherdProvider,
-    estimateGas: true
-  },
-  ubq: {
-    network: 'UBQ',
-    isCustom: false,
-    service: 'ubiqscan.io',
-    lib: shepherdProvider,
-    estimateGas: true
-  },
-
-  exp_auto: {
-    network: 'EXP',
-    isCustom: false,
-    service: 'AUTO',
-    lib: shepherdProvider,
-    estimateGas: true
-  },
-  exp_tech: {
-    network: 'EXP',
-    isCustom: false,
-    service: 'Expanse.tech',
-    lib: shepherdProvider,
-    estimateGas: true
-  },
-  poa_auto: {
-    network: 'POA',
-    isCustom: false,
-    service: 'AUTO',
-    lib: shepherdProvider,
-    estimateGas: true
-  },
-  poa: {
-    network: 'POA',
-    isCustom: false,
-    service: 'poa.network',
-    lib: shepherdProvider,
-    estimateGas: true
-  },
-  tomo_auto: {
-    network: 'TOMO',
-    isCustom: false,
-    service: 'AUTO',
-    lib: shepherdProvider,
-    estimateGas: true
-  },
-  tomo: {
-    network: 'TOMO',
-    isCustom: false,
-    service: 'tomocoin.io',
-    lib: shepherdProvider,
-    estimateGas: true
-  },
-  ella_auto: {
-    network: 'ELLA',
-    isCustom: false,
-    service: 'AUTO',
-    lib: shepherdProvider,
-    estimateGas: true
-  },
-  ella: {
-    network: 'ELLA',
-    isCustom: false,
-    service: 'ellaism.org',
-    lib: shepherdProvider,
-    estimateGas: true
-  }
-};
+import { RawNodeConfig } from 'types/node';
+import { StaticNetworkIds } from 'types/network';
+
+function makeStateFromNodeConfigs(prev: Partial<StaticNodesState>, network: StaticNetworkIds) {
+  // Auto network
+  prev[makeAutoNodeName(network)] = {
+    network,
+    isAuto: true,
+    isCustom: false,
+    service: 'AUTO',
+    estimateGas: NODE_CONFIGS[network].reduce((bool, node) => bool || node.estimateGas, false)
```

Comment:
Do we actually use this property anywhere? Seems like its more suited towards network configs

Reply:
Huh, I had assumed that had something to do with estimating gas limit and that some nodes didn't support it. But you're right, absolutely nothing is using it.